package.json

```diff
@@ -9,6 +9,10 @@
     "search",
     "vanilla"
   ],
+  "engines": {
+    "node": "8.12.0",
```

Comment:
just make sure that this doesn't end up in the package.json that gets pushed to npm, worst case, filter it out before publishing

Reply:
Let's remove it since it doesn't bring value.

Reply:
Not sure to understand your comment @Haroenv. Why it would have an impact on the published package? The [`engines`](https://docs.npmjs.com/files/package.json#engines) attribute will only throw an error when a script is ran on that package specifically otherwise it's only a warning. It will only impact contributors.

@francoischalifour It does bring more value because not everyone is using `nvm`. For those who don't, the attribute is useful. One question remains on the strictness of the value. We can probably use a range rather than a pinned version. It will still enforce a value without bothering too much the contributors.